### ISSUE TITLE
optical flow model using inceptionresnetv2 to train and experience overfit

### ISSUE THREAD

**Issue by ray342659093** [yjxiong/tsn-pytorch#18]
Hi, have you try to train tsn based on inceptionresnetv2 model?
I set the batch size: 8 lr: 0.001 segment num: 3 dropout: 0.8, and then train the rgb model and can achieve 86.32% on test.

however, when I using the same setting with 0.7 dropout to train the optical flow model, and get like 99% training acc but maximal 75% on validation. I think the overfitting occurred. Do you have any idea about the difference performance between the rgb and optical flow model using inceptionresnetv2?

**Comment by yjxiong** [comment]
Please evaluate your flow models using the test scripts. Validation numbers are just for monitoring the training.

**Comment by bryanyzhu** [comment]
@ray342659093 I think you mean testing, right? Because I use ResNet152 and I also experience the same thing as you encountered. Could you share how you overcome the over-fitting? Thanks.

**Comment by ray342659093** [comment]
@bryanyzhu I actually did not fix this problem, instead, I used the resnet101 model, which does not seem to have a severe overfit problem.

**Comment by bryanyzhu** [comment]
@ray342659093 Thanks for sharing.
